.travis.yml

```diff
@@ -39,8 +39,13 @@ jobs:
 
     # Full run of latest supported version, without xdist.
     # Coverage for:
+    # - pytester's LsofFdLeakChecker
+    # - TestArgComplete (linux only)
+    # - numpy
+    # - old attrs
+    # - verbose=0
     # - test_sys_breakpoint_interception (via pexpect).
-    - env: TOXENV=py37-pexpect PYTEST_COVERAGE=1
+    - env: TOXENV=py37-lsof-numpy-oldattrs-pexpect-twisted PYTEST_COVERAGE=1 PYTEST_ADDOPTS=
```

Comment:
based on the diff, this was already included, and the others sort a different  set of things, please elaborate on the details as im a bit puzzled

Reply:
We have one job less - see the commit message (IIRC).